docs/data-sources/vuln.md

```diff
@@ -40,29 +40,29 @@ data "wallarm_vuln" "vulns" {
 
 ## Argument Reference
 
-`filter` - (Required) Filters set in the `key=value` format used to look up for vulnerability details. Possible keys:
+`filter` - (**required**) Filters set in the `key=value` format used to look up for vulnerability details. Possible keys:
 
-- `status` - (Optional) Vulnerability status. Can be: `open` for active vulnerabilities, `closed` for closed vulnerabilities, `falsepositive` for vulnerabilities marked as false positive.
-- `limit` - (Optional) Integer value defining the number of vulnerabilities to return. Possible values: 0-1000.
-- `offset` - (Optional) Integer value defining the number from which vulnerabilities should be returned. Possible values: 0 - 9199999999999999999.
+- `status` - (optional) vulnerability status. Can be: `open` for active vulnerabilities, `closed` for closed vulnerabilities, `falsepositive` for vulnerabilities marked as false positive.
+- `limit` - (optional) integer value defining the number of vulnerabilities to return. Possible values: 0-1000.
+- `offset` - (optional) integer value defining the number from which vulnerabilities should be returned. Possible values: 0 - 9199999999999999999.
 
 ## Attributes Reference
 
-`vulns` - Vulnerability attributes in the `key=value` format. Possible keys:
+`vulns` - vulnerability attributes in the `key=value` format. Possible keys:
 
-- `vuln_id` - Integer vulnerability ID.
-- `wid` - String vulnerability ID.
-- `status` - Vulnerability status. Can be: `open` for active vulnerabilities, `closed` for closed vulnerabilities, `falsepositive` for vulnerabilities marked as false positive.
-- `type` - Vulnerability type (`ptrav`, `sqli`, `infoleak`).
+- `vuln_id` - integer vulnerability ID.
+- `wid` - string vulnerability ID.
+- `status` - vulnerability status. Can be: `open` for active vulnerabilities, `closed` for closed vulnerabilities, `falsepositive` for vulnerabilities marked as false positive.
+- `type` - vulnerability type (`ptrav`, `sqli`, `infoleak`).
```

Comment:
there are many other types

Reply:
@aoleynikov-wallarm, can we add "and others" with the link to https://docs.wallarm.com/attacks-vulns-list/ or only these 3 listed are supported?

Reply:
Yes, we absolutely can.